### DISCUSSION TITLE
Players and Rosters

### DISCUSSION THREAD

**Discussion by slaven711** [kyleskom/NBA-Machine-Learning-Sports-Betting#315] [General]
I ran this for today's games (Oct 25) and noted the models absolutely love the Grizzlies to beat the Pelicans, something like 73%. But the odds are pretty close to even. Both teams have similar expected win totals for the year, so it seems odd the edge/difference vs. the betting line would be so huge. 

Is it possible the model thinks Ja Morant is playing, despite him being suspended? How would go about checking this within the code and/or databases?

**Comment by kyleskom** [comment]
Ya this doesn't take into account players who are injured or not playing.

**Reply by slaven711** [reply]
ok thanks-- but it does pick up the expected value from a player that played on a different team last year? so for example it will pick up that the Bucks should be better this year with Damian Lillard (assuming he is of greater value than the players they dealt away).

**Reply by kyleskom** [reply]
With time yes. It looks at team stats, so as a few games get going those player stats will trickle into the teams stats.
